### PR TITLE
Edits for week of Dec 2

### PR DESCRIPTION
**Pull request by duglin**
- Reorder "defaultversion*" attrs: id, url, sticky - based on importance
- Make it clear that ?nested isn't needed when `model` or `meta` are present
  on requests
- epoch/modifiedat are only updated when nested entities are added or
  removed from the collection, not when just edited
- added `?` before each query parameter (eg `filter`->`?filter`) for consistency

### CODE REVIEW

File: core/spec.md [hunk 1]
```diff
@@ -5127,12 +5137,12 @@ to have any inlinable attributes (except for `RESOURCE*`) that are present in
...
 
-Note: If the Registry can not return all expected data in one response then it
-MUST generate an HTTP `406 Not Acceptable` error and SHOULD include a error
-message in the HTTP body indicating that the response is too large to be
-sent in one message. In those cases, the client will need to query the
-individual inlinable attributes in isolation so the Registry can leverage
-[pagination](../pagination/spec.md) of the response.
+Note: If the Registry can not return all expected data in one response because
+it is too large then it MUST generate an HTTP `406 Not Acceptable` error and
+SHOULD include a error message in the HTTP body indicating which attributes
```
Comment (deissnerk) [inline core/spec.md:5142]:
```suggestion
SHOULD include an error message in the HTTP body indicating which attributes
```

**Review by deissnerk — COMMENTED** [review]
LGTM
As you consistently introduced the `?queryParameter` notation, it might be useful to add a sentence to the _Notational Conventions_ section explaining that query parameters are always highlighted with a leading `?`. WDYT?

File: core/spec.md (hunk 1 above)
Comment (duglin) [reply core/spec.md]: Fixed the "a" and added some text about ?query

**Comment by duglin** [comment]
Approved on the 12/5 call

**Pull request merged**